app/models/course.rb

```diff
@@ -5,6 +5,53 @@
 
 class Course < ActiveRecord::Base
   include SystemCommands
+  include Swagger::Blocks
+
+  swagger_schema :Course do
```

Comment:
It's gonna get really ugly if we're going to include all this swagger logic on most of the models. Should we isolate this into its own file?